site-src/implementations.md

```diff
@@ -129,9 +129,10 @@ Documentation to deploy and use AKO Gateway API can be found at [Avi Kubernetes
 
 ### Azure Application Gateway for Containers
 
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.0.0-AzureApplicationGatewayForContainers-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/azure-application-gateway-for-Containers)
```

Comment:
Not really any prior art here, but maybe something like this would work to to communicate the partial conformance state? (cc @mlavacca @youngnick for second opinion)

```suggestion
[![Conformance](https://img.shields.io/badge/Gateway%20API%20Partial%20Conformance%20v1.0.0-Azure%20Application%20Gateway%20for%20Containers-orange)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/azure-application-gateway-for-containers)
```